language/stdlib/modules/LibraConfig.move

```diff
@@ -164,6 +167,27 @@ module LibraConfig {
         include SetEnsures<Config>;
     }
 
+    fun disable_reconfiguration(lr_account: &signer) {
+        assert(
+            Signer::address_of(lr_account) == CoreAddresses::LIBRA_ROOT_ADDRESS(),
+            Errors::requires_address(ELIBRA_CONFIG)
+        );
+        Roles::assert_libra_root(lr_account);
+        move_to(lr_account, DisableReconfiguration {} )
```

Comment:
Would it make more sense for this to be a field of the existing `Configuration` resource? That seems easier than trying to create a new singleton resource under LIBRA_ROOT.

Reply:
My original thought is that we would like to avoid breaking changes like this. The current PR will leave the on-chain state totally unchanged. I'm a bit more confortable with this approach as we don't have additional unused data on chain.

Reply:
Actually after playing with the code, it is not possible for us to put it in `Configuration`. The reason is that changes to `Configuration` during the WriteSet generation process will cause Configuration resource to be part of that writeset, even if we change the value back.